CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.7.2
+
+* Update `createMiddleware` arguments to use `FutureOr`.
+
+  * Note: this change is not breaking for the runtime behavior, but it might
+    cause new errors during static analysis due the the type changes.
+
+* Updated minimum Dart SDK to `1.24.0`, which added `FutureOr`.
```

Comment:
FYI: We generally don't list dependency range changes (including SDK constraint changes) in the changelog. Pub will handle these automatically, which means making the downstream consumer think about them adds cognitive overhead for little value.

Reply:
...I always thought it can help users understand why they are not getting the latest version.

Reply:
I don't think most users have a workflow that involves looking at their dependencies' changelogs when trying to figure out the version solver's behavior. Once the new version solver lands, they should be able to just run `pub upgrade $package` and get good diagnostics if the latest version isn't available.

Reply:
Can't wait!